evm/gateway.js

```diff
@@ -183,6 +195,38 @@ async function processCommand(config, chain, options) {
             break;
         }
 
+        case 'approveWithBatch': {
+            const { batchID, api } = options;
+
+            if (!batchID) {
+                throw new Error('Batch ID is required for the approve action');
+            }
+
+            const batchId = batchID.startsWith('0x') ? batchID.substring(2) : batchID;
+            let apiUrl = api || `${config.axelar.lcd}/axelar/evm/v1beta1/batched_commands/${chain.name.toLowerCase()}`;
```

Comment:
`chain.axelarId`. use axelar registered chain names when interacting with the protocol